filter.go

```diff
@@ -18,17 +18,9 @@ func NewContainerFilter(names []string) ContainerFilter {
 type containerFilter []string
 
 func (cf containerFilter) Accept(cs v1.ContainerStatus) bool {
-	if !cs.Ready {
-		return false
-	}
-	if len(cf) == 0 {
+	if cs.State.Running != nil {
 		return true
 	}
-	for _, name := range cf {
-		if name == cs.Name {
-			return true
-		}
-	}
```

Comment:
Just replace the `!cs.Ready` expression; the rest is used for filtering by container name with `--container`.

Reply:
i won’t be near a computer until monday though, so it will have to wait sadly, but feel free to overtake this pr and put it back in, if you’d like, otherwise i’ll followup next week :)